src/gui/transferlistdelegate.cpp

```diff
@@ -33,16 +34,10 @@
 #include "transferlistmodel.h"
 
 TransferListDelegate::TransferListDelegate(QObject *parent)
-    : QStyledItemDelegate {parent}
+    : QStyledItemDelegate(parent)
```

Comment:
@glassez 
Off topic, what exactly is the rule of using `{}` or `()` at this place (and for variable initialization)? I can't figure out your pattern and it looks like changing back and forth.

Reply:
>We use {} after variable name and () after constructor name (whenever possible).

OK. Pardon me that I don't remember it.
Were there any effective reasons for it or it was just stylistic?